crates/curp/src/log_entry.rs

```diff
@@ -23,6 +23,8 @@ pub struct LogEntry<C> {
     pub(crate) propose_id: ProposeId,
     /// Entry data
     pub(crate) entry_data: EntryData<C>,
+    /// entry size
+    pub(crate) size: u64,
```

Comment:
I suggest to manually implement the Serialize and Deserialize trait for LogEntry, other wise the size field could occupy some space.

Reply:
I move the size field out of LogEntry and wrap it with entry in a new Entry struct.